packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -52,6 +53,7 @@
 	{@param? handleViewTypeClicked_: any}
 	{@param? hideFiltersDoneButton: bool}
 	{@param? id: string}
+	{@param? infoButton: bool}
```

Comment:
For consistency with the rest of the params I think this should be named `showInfoButton`